bundles/org.openhab.binding.openweathermap/README.md

```diff
@@ -322,12 +322,12 @@ DateTime localLastMeasurement "Timestamp of Last Measurement [%1$tY-%1$tm-%1$tdT
 DateTime localTodaySunrise "Todays Sunrise [%1$tY-%1$tm-%1$tdT%1$tH:%1$tM:%1$tS]" <time> { channel="openweathermap:onecall:api:local:current#sunrise" }
 DateTime localTodaySunset "Todays Sunset [%1$tY-%1$tm-%1$tdT%1$tH:%1$tM:%1$tS]"  <time> { channel="openweathermap:onecall:api:local:current#sunset" }
 String localCurrentCondition "Current Condition [%s]" <sun_clouds> { channel="openweathermap:onecall:api:local:current#condition" }
-Image localCurrentConditionIcon "Icon" { channel="openweathermap:onecall:api:local:current#Icon" }
+Image localCurrentConditionIcon "Icon" { channel="openweathermap:onecall:api:local:current#icon" }
 Number:Temperature localCurrentTemperature "Current Temperature [%.1f %unit%]" <temperature> { channel="openweathermap:onecall:api:local:current#temperature" }
 Number:Temperature localCurrentApparentTemperature "Current Apparent Temperature [%.1f %unit%]" <temperature> { channel="openweathermap:onecall:api:local:current#apparent-temperature" }
 Number:Pressure localCurrentPressure "Current barometric Pressure [%.1f %unit%]" <pressure> { channel="openweathermap:onecall:api:local:current#pressure" }
 Number:Dimensionless localCurrentHumidity "Current atmospheric Humidity [%d %unit%]" <humidity> { channel="openweathermap:onecall:api:local:current#humidity" }
-Number:Temperature localCurrentDewpoint  "Current Dew-point [%.1f %unit%]" <Temperature> { channea="openweathermap:onecall:api:local:eurrent#dew-point" }
```

Comment:
Should we may be align the label too? "Current Dew point" without the "-".

Reply:
yes, seems to be better to do so....